### PR TITLE
Item Duplication FIX

### PR DESCRIPTION
**Pull request by ak4y**
Dupe fix 

https://github.com/benlynx/qb-inventory/commit/743e3b5b39e55d6d3889c5777f7be30a6f9f64dc

### CODE REVIEW

**Comment by BerkieBb** [comment]
Are you able to show a comparison of what this fixes?

**Comment by benlynx** [comment]
> Are you able to show a comparison of what this fixes?

You can check this 
https://github.com/qbcore-framework/qb-inventory/pull/263/commits/743e3b5b39e55d6d3889c5777f7be30a6f9f64dc

**Comment by Ruqen** [comment]
Still needs some formatting. Printing values/errors is not a good idea, please fix the prints according to the logs.

File: server/main.lua [hunk 1]
```diff
@@ -959,10 +972,14 @@ RegisterNetEvent('inventory:server:SetInventoryData', function(fromInventory, to
...
-						Player.Functions.AddItem(toItemData.name, toAmount, fromSlot, toItemData.info)
-						TriggerEvent("qb-log:server:CreateLog", "glovebox", "Swapped Item", "orange", "**".. GetPlayerName(src) .. "** (citizenid: *"..Player.PlayerData.citizenid.."* | id: *"..src.."*) swapped item; name: **"..itemInfo["name"].."**, amount: **" .. toAmount .. "** with name: **" .. fromItemData.name .. "**, amount: **" .. fromAmount .. "** - plate: *" .. plate .. "*")
+					if toItemData.amount >= toAmount then
+						if toItemData.name ~= fromItemData.name then
+							RemoveFromGlovebox(plate, fromSlot, itemInfo["name"], toAmount)
+							Player.Functions.AddItem(toItemData.name, toAmount, fromSlot, toItemData.info)
+							TriggerEvent("qb-log:server:CreateLog", "glovebox", "Swapped Item", "orange", "**".. GetPlayerName(src) .. "** (citizenid: *"..Player.PlayerData.citizenid.."* | id: *"..src.."*) swapped item; name: **"..itemInfo["name"].."**, amount: **" .. toAmount .. "** with name: **" .. fromItemData.name .. "**, amount: **" .. fromAmount .. "** - plate: *" .. plate .. "*")
+						end
+					else
+						print("DUPE ENGELLENDI - 4")
```
Comment (restray) [inline server/main.lua:982]: I love the way you're making your prints lmao! But I do not understand this one at all!

Replace it with a createLog please :)

**Review by restray — CHANGES_REQUESTED** [review]
I didn't have the time to test it with a live server but your work is impressive!
Thanks for your time, just need 2 or 3 changes :)

**Review by Txc32 — APPROVED** [review]

**Comment by ChickenDipper1** [comment]
Surely this deserves more attention! I'll try this myself, thank you!

**Comment by Z3rio** [comment]
Abandoned PR and incorrect usage of PR description / not following template

**Pull request closed without merging**